js/ui/date_range_box.js

```diff
@@ -0,0 +1,34 @@
+import registerComponent from '../core/component_registrator';
+import { extend } from '../core/utils/extend';
+import DateBox from './date_box';
+
+const DateRangeBox = DateBox.inherit({
+    _getDefaultOptions: function() {
+        return extend(this.callBase(), {
+            /**
+            * @name dxDateRangeBoxOptions.adaptivityEnabled
+            * @hidden
+            */
+
+            /**
+            * @name dxDateRangeBoxOptions.interval
+            * @hidden
+            */
+
+            /**
+             * @name dxPopoverOptions.showAnalogClock
+             * @hidden
+             */
+            showAnalogClock: false,
+
+            /**
+            * @name dxDateRangeBoxOptions.type
+            * @hidden
+            */
```

Comment:
Cannot we hide all of this in d.ts, not here?

Reply:
there's literally 0 (zero) options that are hidden in d.ts file in whole project. We discussed this once. I think it's not for some some kind of joke, it's made in purpose :)

@IlyaKhD what is the reason of hiding options in .js instead of .d.ts?

Reply:
Keep it as is for now